include/networkit/algebraic/CSRMatrix.hpp

```diff
@@ -1,612 +0,0 @@
-/*
- * CSRMatrix.h
- *
- *  Created on: May 6, 2015
- *      Author: Michael Wegner (michael.wegner@student.kit.edu)
- */
```

Comment:
We should not delete the entire header file. This requires quite a bit of refactoring in NetworKit and it might also require users to adopt. Instead, keep `CSRMatrix.hpp` header, let it include the `CSRGeneralMatrix.hpp` header and move the typedef to the `CSRMatrix.hpp` header.